BZ1973326: Add 4-8 RN for multipath (#34798)
openshift/openshift-docs: release_notes/ocp-4-8-release-notes.adoc

If a cloud administrator has already set a custom `/etc/chrony.conf` configuration, {op-system} no longer sets the `PEERNTP=no` option by default on cloud platforms. Otherwise, the `PEERNTP=no` option is still set by default. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1924869[BZ#1924869] for more information.

[id="ocp-4-8-rhcos-"multipath-install"]
==== Enabling multipathing at installation

> Enabling multipathing during installation is now supported and recommended for nodes provisioned in {product-title} 4.8 or higher. You can enable multipathing by appending kernel arguments to the `coreos-installer install` command so that the installed system itself will use multipath beginning from the first boot.

Reviewer: Is there any platform restriction on multipath support? Or do we actually recommend this for all platforms?

Author: > Is there any platform restriction on multipath support? > Or do we actually recommend this for all platforms?

Hi @miabbott - My understanding is that it would be recommended for bare metal, so I've clarified that in the wording. Is this correct, @jlebon?

Also, I've clarified the wording so it doesn't sound like _all_ clusters should have multipath enabled. The main point is that we recommend multipathing at install over configuring it as a day-2 operation, not as a general principle.

If a cloud administrator has already set a custom `/etc/chrony.conf` configuration, {op-system} no longer sets the `PEERNTP=no` option by default on cloud platforms. Otherwise, the `PEERNTP=no` option is still set by default. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1924869[BZ#1924869] for more information.

> [id="ocp-4-8-rhcos-"multipath-install"]

Reviewer 2: ```suggestion
[id="ocp-4-8-rhcos-multipath-install"]
```

Author: Good catch, thanks!

==== Enabling multipathing at bare metal installation time

> Enabling multipathing during bare metal installation is now supported for nodes provisioned in {product-title} 4.8 or higher. You can enable multipathing by appending kernel arguments to the `coreos-installer install` command so that the installed system itself will use multipath beginning from the first boot. While post-installation support is still available by activating multipathing via the machine config, enabling multipathing during installation is recommended for nodes provisioned starting in {product-title} 4.8.

Reviewer 2: ```suggestion
Enabling multipathing during bare metal installation is now supported for nodes provisioned in {product-title} 4.8 or higher. You can enable multipathing by appending kernel arguments to the `coreos-installer install` command so that the installed system itself uses multipath beginning from the first boot. While post-installation support is still available by activating multipathing through the machine config, enabling multipathing during installation is recommended for nodes provisioned starting in {product-title} 4.8.
```

> ! via preposition
Use this term, which means "by way of", to refer to routing data, such as in "Messages are routed via SMTP". In other situations, try to replace "via" with one of the following synonyms: "across", "along", "by", "from", "on", or "through".

Author: Thanks, @kalexand-rh.

- I contemplated "uses" rather than "will use" in this particular case because it seemed a bit of a sequential process: append kargs now so that it will be used at first boot. I was probably overthinking it, and "uses" has the benefit of sounding fine and also conforming to ISG. Changed.
- I also contemplated replacing "via" but nothing sounded more appropriate in this case. IMO, "activating multipathing _through_ the machine config" makes it sound like it's actually traveling through the MC, which seems odd. In this occurrence, I take it to actually mean "by way of", so it seems "via" is the most appropriate. Unchanged.
